src/DataGrid.tsx

```diff
@@ -196,16 +188,13 @@ function DataGrid<R, SR, K extends Key>(
     // Dimensions props
     rowHeight,
     headerRowHeight,
```

Comment:
To fix `headerRowHeight` being potentially nullish.
```suggestion
    headerRowHeight: rawHeaderRowHeight,
```